Gruntfile.coffee

```diff
@@ -201,38 +201,38 @@ module.exports = (grunt) ->
     
     
   
-    dom_munger:
-      read: 
-        options: 
-          read:[
-            {selector:'script[in-production!="false"]', attribute:'src', writeto:'appjs'}
-            {selector:'link[rel="stylesheet"][in-production!="false"]', attribute:'href', writeto:'appcss'}
-          ]
-        
-        src: 'src/index.html'
-      
-      update: 
-        options: 
-          remove: ['script', 'link[rel="stylesheet"]']
-          append: [
-            {selector: 'body', html:'<script src="weaver.js"></script>'}
-            {selector:'head', html:'<link rel="stylesheet" href="weaver.css">'}
-          ]
-        
-        src:'src/index.html'
-        dest: 'dist/index.html'
+#    dom_munger:
+#      read:
+#        options:
+#          read:[
+#            {selector:'script[in-production!="false"]', attribute:'src', writeto:'appjs'}
+#            {selector:'link[rel="stylesheet"][in-production!="false"]', attribute:'href', writeto:'appcss'}
+#          ]
+#
+#        src: 'src/index.html'
+#
+#      update:
+#        options:
+#          remove: ['script', 'link[rel="stylesheet"]']
+#          append: [
+#            {selector: 'body', html:'<script src="weaver.js"></script>'}
+#            {selector:'head', html:'<link rel="stylesheet" href="weaver.css">'}
+#          ]
+#
+#        src:'src/index.html'
+#        dest: 'dist/index.html'
```

Comment:
Just delete these lines, version control will remember them if they are needed again in the future